README.md

```diff
@@ -409,6 +409,7 @@ Languages supported:
 * Turkish ('tr')
 * Ukrainian ('uk')
 * Vietnamese ('vi')
+* Kurdish Sorani ('ku')
```

Comment:
@drpshtiwan Thanks for this PR. The  Kurdish Sorani ('ku') language should go after Korean ('ko') in all places because they are sorted alphabetically.